<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>307</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>237</width>
    <height>307</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>237</width>
    <height>307</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dsSaveScene</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="project_L">
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Project:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="projects_CB">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="episode_L">
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Episode:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="episodes_CB">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="seq_L">
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sequence:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="sequence_CB">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="task_L">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Task:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="task_CB">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="fn_L">
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>fileName:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="fn_LE">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="episode_L_2">
      <property name="maximumSize">
       <size>
        <width>201</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Notation:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLineEdit" name="nn_LE">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="save_B">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
